exist-core/src/test/xquery/dates/convert-dates.xqm

```diff
@@ -25,68 +25,68 @@ module namespace fd="http://exist-db.org/xquery/test/convert-dates";
 
 declare namespace test="http://exist-db.org/xquery/xqsuite";
 
-(: verify that we can convert xs:string to :xs:dateTimeStamp  :)
+(: verify that we can convert xs:string to :xs:dateTime :)
 declare
     %test:args("2012-06-26T23:14:22.566+02:00")
     %test:assertEquals("11:14 pm on Tuesday, June 26th, 2012")
-function fd:format-dateTimeStamp($date as xs:dateTimeStamp) {
+function fd:format-dateTime($date as xs:dateTime) {
     format-dateTime($date, "[h00]:[m00] [P] on [FNn], [MNn] [D1o], [Y]", "en", (), ())
 };
 
-(: verify that we can't convert xs:string without timezone to :xs:dateTimeStamp  :)
+(: verify that we can't convert xs:string without timezone to :xs:dateTime :)
 declare
     %test:args("2012-06-26T23:14:22.566")
     %test:assertError
-function fd:convert-date-time-stamp($date as xs:dateTimeStamp) {
+function fd:convert-date-time($date as xs:dateTime) {
     format-dateTime($date, "[h00]:[m00] [P] on [FNn], [MNn] [D1o], [Y]", "en", (), ())
 };
 
 declare
     %test:args("2022-05-17T16:24:06.003+02:00", "PT2H")
     %test:assertEquals("2022-05-17T18:24:06.003+02:00")
-function fd:add-test($date as xs:dateTimeStamp, $duration as xs:dayTimeDuration) {
+function fd:add-test($date as xs:dateTime, $duration as xs:dayTimeDuration) {
     $duration + $date
 };
 
 
-(: verify that fn:current-dateTime() return type is xs:dateTimeStamp  :)
+(: verify that fn:current-dateTime() return type is xs:dateTime  :)
 declare
     %test:assertEquals("true")
 function fd:current-dateTime-type() {
-    fn:current-dateTime() instance of xs:dateTimeStamp
+    fn:current-dateTime() instance of xs:dateTime
```

Comment:
As with my comment above, this change should be undone.

Reply:
This has also now been undone.